Assets/Scripts/World/Units/Enemy/Main/EnemyController.cs

```diff
@@ -72,6 +74,7 @@ public class EnemyController : UnitController<IEnemy, Enemy, EnemyList>, IEnemyC
 		if (orderedEnemyPrefabs.ContainsKey(enemyKey)) {
 			GameObject enemy = (GameObject) Instantiate(orderedEnemyPrefabs[enemyKey], EnemySpawnPoint.GetPosition(spawnDirection), Quaternion.identity);
 			EnemyBehaviour enemyBehaviour = enemy.GetComponent<EnemyBehaviour>();
+            AddActiveEnemy(enemyBehaviour);
```

Comment:
This looks good. The only note I have is that you'll also want to be calling "RemoveActiveEnemy" inside EnemyBehaviour when the enemy gets destroyed